src/desktop_develop.ts

```diff
@@ -50,23 +73,23 @@ function getNextBuildTime(d) {
     return next;
 }
 
-async function getLastBuildTime(type) {
+async function getLastBuildTime(type: Type): Promise<number> {
     try {
-        return await fsProm.readFile('desktop_develop_lastBuilt_' + type, 'utf8');
+        return parseInt(await fsProm.readFile('desktop_develop_lastBuilt_' + type, 'utf8'));
     } catch (e) {
         return 0;
     }
 }
 
-async function putLastBuildTime(type, t) {
+async function putLastBuildTime(type: Type, t: number): Promise<void> {
     try {
-        return await fsProm.writeFile('desktop_develop_lastBuilt_' + type, t);
+        return await fsProm.writeFile('desktop_develop_lastBuilt_' + type, t.toString());
```

Comment:
it's a little bit awkward to have a return like this on a `void` return type, but not unheard of.

Reply:
Sure, have changed.